### PR TITLE
First basic schedule

### PR DESCRIPTION
**Pull request by badboy**
- [ ] Link from homepage
- [x] Ashley's talk title
- [x] Design changes?
- [ ] [Full announcement](https://github.com/RustFestEU/rustfest.eu/pull/46)

Current design: 
fireshot capture 3 - schedule - http___localhost_4000_schedule_

### CODE REVIEW

**Comment by dns2utf8** [comment]
How long will the last event "The Rust that could have been" be?

EDIT: I would capitalize it like the others, which seems to be the german way :+1:

**Comment by Hoverbear** [comment]
Design looks good. +1 on that.

**Comment by gnunicorn** [comment]
@badboy can I has more margin between the last event of day one and the title of day two? At least 1.5em.

**Comment by badboy** [comment]
Yes, yes you can.

File: schedule.html [hunk 1]
```diff
@@ -88,7 +88,7 @@ <h2 class="title">Saturday, 17th September</h2>
         <td class="time">13<span>30</span></td>
         <td>
           <a href="http://www.rustfest.eu/talks/">
-            TBA
+            intermezzOS: a teaching operating system
```
Comment (Hoverbear) [inline schedule.html:91]: 👐

File: schedule.html [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+          </a>
+          <div class='speaker'>
+            <img src="/assets/speakers/martin.jpg" />
+            Martin Hellspong
+          </div>
+        </td>
+      </tr>
+      <tr>
+        <td class="time">15<span>00</span></td>
+        <td class="centered">Break</td>
```
Comment (gnunicorn) [inline schedule.html]: @badboy : NEW – there will be coffee and cake here!

**Pull request merged**